src/live.asp.net/wwwroot/js/site.js

```diff
@@ -80,11 +80,22 @@
         }
     }
 
-    function countdownTo(futureDate, tickCallback, endCallback) {
+    function countdownTo(futureDate, isOnAirPolling, tickCallback, endCallback) {
         var interval = window.setInterval(function () {
             var now = new Date(),
                 diff = dateDiff(now, futureDate);
 
+			if (!isOnAirPolling && diff.totalSecs < 3 * 60) {
```

Comment:
Might be nice to put these numbers in well-named variables so it's obvious what it is, e.g. `startPollingForLiveShow`